packages/api-plugin-email/src/startup.js

```diff
@@ -1,12 +1,15 @@
-import processEmailJobs from "./util/processEmailJobs.js";
+import returnEmailProcessor from "./util/returnEmailProcessor.js";
+import config from "./config.js";
 
+const { REACTION_WORKERS_ENABLED } = config;
 /**
  * @summary Called on startup
  * @param {Object} context Startup context
  * @param {Object} context.collections Map of MongoDB collections
  * @returns {undefined}
  */
-export default function emailStartup(context) {
-  processEmailJobs(context);
+export default async function emailStartup(context) {
+  if (!REACTION_WORKERS_ENABLED) return;
```

Comment:
I would handle these `REACTION_WORKERS_ENABLED` slightly differently. What if you don't want this instance working jobs, but you do still need to schedule emails to be sent, etc.? You'll need the queue to exist to add jobs to it.

You could:
- Make `processorFn` arg optional
- Don't pass `processorFn` when `!REACTION_WORKERS_ENABLED`
- In `createQueue`, skip the `.process` call when `processorFn` isn't provided

I'm not sure which way I lean, but you might also consider respecting `REACTION_WORKERS_ENABLED` env directly in the bull plugin. That way each consumer plugin wouldn't have to remember to separately respect it.

Reply:
So I am basically taking the 
> Don't pass processorFn when !REACTION_WORKERS_ENABLED

In createQueue so that you can add jobs to your queue, but no processor is attached.